### DISCUSSION TITLE
ERROR: Please reload the website you are on and then use the extension.

### DISCUSSION THREAD

**Discussion by caesar5877** [GorvGoyl/ChatGPT-Writer#45] [Report Bugs and Issues]
I installed it and click icon then show this error messsage. 
I don't see icon in gmail either. (I restarted my laptop )
![Snip20230203_1](https://user-images.githubusercontent.com/3526175/216726080-d4a4ebe0-8d0b-4a48-aedb-536d1876b01b.png)

**Comment by caesar5877** [comment]
**UPDATE:::** 
update Google Chrome to the latest version, then chatGPT writer works!!

**Reply by telbahnasawy** [reply]
I have the latest Chrome version "Version 110.0.5481.77 (Official Build) (x86_64)" and still getting this same error.

**Reply by Jillu-Patel** [reply]
I am getting same error.
I have updated chrome

**Comment by Jillu-Patel** [comment]
Same issue 
please Let me know if some one finds the solution

**Comment by kashimotoxiang** [comment]
+1
